packages/ui/src/deprecated/ContainerSpinner.tsx

```diff
@@ -0,0 +1,18 @@
+import { useClassName } from '@contember/utilities'
+import { memo } from 'react'
+import { SpinnerContainerProps, SpinnerOverlay } from '../components'
+
+/**
+ * @group UI
+ * @deprecated use SpinnerOverlayProps instead
+ */
+export type ContainerSpinnerProps = SpinnerContainerProps
+
+/**
+ * @group UI
+ * @deprecated use SpinnerOverlay instead
+ */
+export const ContainerSpinner = memo((props: ContainerSpinnerProps) => (
```

Comment:
do we really need to move it to `depracated` entrypoint? it is used in userland, so I would prefer soft-deprecation first

Reply:
What is not soft-deprecated about this?

Reply:
Done